src/delaunay_diagram.py

```diff
@@ -7,16 +8,16 @@ class DelaunayDiagram:
      - neighbours: dictionary that for each point holds a set of points that are its neighbours
      - neighbours_making_triangles: for each neighbouring points returns a set of points that make are common neighbour
        with the two making the key
-     - triangles: a set of tuples of points making triangles
+     - triangles: a dictionary that for tuples of points contains triangle areas
```

Comment:
Pytanie - czy nie lepiej byłoby przechowywać zbiór punktów zamiast krotki? Może pozwoliłoby to uniknąć przypisywania tej samej powierzchni 6 razy.

Reply:
byłoby, niestety nie jest to technicznie możliwe - zbiór nie jest hashowalny więc nie może być kluczem w słowniku